Modif LCD & PCP:

medzila opened the pull request:
+Laser entre le drogué et le drogueur
+On ne peut plus droguer en zone safe
+Affichage de messages pour le drogueur et le drogué
+Factorisation du code

medzila committed:
Modif LCD & PCP:
+Laser entre le drogué et le drogueur
+On ne peut plus droguer en zone safe
+Affichage de messages pour le drogueur et le drogué
+Factorisation du code

kossolax commented:
Qql changement que je te laisse faire:

L67, L97 il manque un )
L93 changer par target
L101+L102 attention à l'indentation 

 Merci à toi !

medzila committed:
Modif LCD & PCP(corrigé):
+Laser entre le drogué et le drogueur
+On ne peut plus droguer en zone safe
+Affichage de messages pour le drogueur et le drogué
+Factorisation du code

medzila committed:
Modif LCD & PCP (indent?):
+Laser entre le drogué et le drogueur
+On ne peut plus droguer en zone safe
+Affichage de messages pour le drogueur et le drogué
+Factorisation du code

medzila committed:
Modif LCD & PCP (corrigé)
+Laser entre le drogué et le drogueur
+On ne peut plus droguer en zone safe
+Affichage de messages pour le drogueur et le drogué
+Factorisation du code

medzila committed:
Ajout du "Pack Debutant"
+contient la CB, le compte et le RIB
+corrections mineures de fautes
